### DISCUSSION TITLE
How to Run SuperSet in PyCharm

### DISCUSSION THREAD

**Discussion by codegeekjk** [apache/superset#13379] [Q&A / Help]
Can someone please guide me how can I run superset frontend in pycharm ?

For Backend I have created a file `run.py` with the following contents 
```
from superset import create_app

if __name__ == '__main__':
    app = create_app()
    app.run(host="localhost", port=8080, debug=True)
```

How to run the frontend code so I can start the deployment process ?

**Comment by codegeekjk** [comment]
Update: I am run the frontend using` npm run dev-server`, I am not able to figure out how can  put debug point

**Reply by etr2460** [reply]
you can add debuggers in the javascript code with `debugger;` and then when your chrome devtools is open it should pause code execution and you'll be able to step through the code. Not sure if this is what you're asking though
